lib/charms/archive_auth_mirror/utils.py

```diff
@@ -6,7 +6,7 @@
 from charmhelpers.core import host
 
 
-def get_paths(base_dir=None):
+def get_paths(root_dir=Path('/')):
```

Comment:
Maybe defaulting to None would be better? Path('/') is probably immutable, but it feels safer to use None.